lib/AbstractHandle/Utils/MongoUtil.py

```diff
@@ -104,7 +105,7 @@ def find_in(self, elements, field_name, projection={'_id': False}, batch_size=10
                             ''.join(traceback.format_exception(None, e, e.__traceback__)))
             raise ValueError(error_msg)
 
-        logging.info('returned {} results'.format(result.count()))
+        logging.info('returned {} results'.format(len(list(result.clone()))))
```

Comment:
This will now load the entire result set into memory rather than a single integer

Reply:
IIUC clone() correctly it also means the query is resubmitted

Reply:
I don't think the query will be resubmitted. It only clones the cursor.

Reply:
https://pymongo.readthedocs.io/en/stable/api/pymongo/cursor.html#pymongo.cursor.Cursor.clone

> The clone will be completely unevaluated, even if the current instance has been partially or completely evaluated.

Reply:
Not sure if there is a better way to do this besides converting `result` to a list directly.

Reply:
Maybe the best choice is to not do it at all. Can we just delete that line?

Reply:
Yes, we can because here is just logs.
I also have a bunch of `len(list(result.clone())))` in MongoUtil_test.py. I don't think we can just delete them there.

Reply:
If it's just test code I'm not worried about it. Loading the entire result set into memory here could potentially oom the service or reduce performance

Reply:
I see.

Reply:
👍

Reply:
:+1: